android/src/lib.rs

```diff
@@ -23,7 +23,15 @@ pub fn android_main() {
 
 #[no_mangle]
 pub extern "system" fn Java_org_maplibre_1rs_MapLibreRs_android_1main(env: JNIEnv, class: JClass) {
-    let tag = CString::new("maplibre").unwrap();
-    let message = CString::new("maplibre WOORKING").unwrap();
+    let tag = match CString::new("maplibre");
+    let tag = match tag {
+        Ok(tagg) => tagg,
+        Err(e) => (e),
+    };
+    let message = CString::new("maplibre WOORKING");
+    let message = match message {
+        Ok(mess) => mess,
+        Err(e) => (e),
+    };
```

Comment:
I'm quite sure this won't compile because there is an unneccassary `match` in the first line.
Also the return types of the match arms do not match.

This use if `.unwrap` also happens in a dummy-example used to test JNI for android. This code will eventually be removed soonish.
The general idea was to replace uses of `unwrap` by propagating the error type.